### PR TITLE
Fix some formatting and get a very basic gRPC server running/working

### PR DESCRIPTION
**Pull request by iron768**
Start working on gRPC and do some light formatting for the repository. We will attempt to finish the gRPC implementation in this PR. Me and @lettucegoblin and @ndg8743 did pair programming for this PR.

### CODE REVIEW

File: src/main/proto/test.proto [hunk 1]
```diff
@@ -1,5 +1,7 @@
 // test.proto
-message test {
+package org.fibsters;
```
Comment (lettucegoblin) [inline src/main/proto/test.proto]: Delete the test file

**Review by lettucegoblin — CHANGES_REQUESTED** [review]

File: src/main/proto/test.proto (hunk 1 above)
Comment (iron768) [reply src/main/proto/test.proto]: Deleted!

File: src/main/java/org/cliclient/CliClient.java [hunk 2]
```diff
@@ -34,27 +47,78 @@ public class CliClient {
...
-        while (true) {
+            System.out.println("Set Network Request Type (POST/GRPC), POST by default");
+            String response = scanner.nextLine();
+            if ("GRPC".equalsIgnoreCase(response)) {
+                CliClient.networkRequestType = "GRPC";
+            }
 
-            List<Integer> userInput = getUserInputArr();
-            int[] userinputArr = userInput.stream().mapToInt(i -> i).toArray();
+            System.out.println("Set file name (Don't specify file type!): ");
```
Comment (lettucegoblin) [inline src/main/java/org/cliclient/CliClient.java:65]: We'll definitely want to specify if it's locally or server side and make the images available server side at those endpoint filenames.

**Review by lettucegoblin — APPROVED** [review]
Looking good!

**Pull request merged**